client/packages/system/src/Patient/Encounter/CreateEncounterModal.tsx

```diff
@@ -33,6 +34,47 @@ import {
   ClinicianSearchInput,
 } from '../../Clinician';
 import { PatientTabValue } from '../PatientView/PatientView';
+import { PickersDay, PickersDayProps } from '@mui/x-date-pickers';
+import Badge from '@mui/material/Badge';
+
+type HighlightedDay = { datetime: Date; label?: string | null };
+type BadgePickersDayProps = {
+  highlightedDays: HighlightedDay[];
+};
+const BadgePickersDay = (
+  props: PickersDayProps<Date> & BadgePickersDayProps
+) => {
+  const { highlightedDays = [], day, outsideCurrentMonth, ...other } = props;
+
+  const matchingDay = highlightedDays.find(it => {
+    const date = it.datetime;
+    return (
+      day.getFullYear() === date.getFullYear() &&
+      day.getMonth() === date.getMonth() &&
+      day.getDate() === date.getDate()
+    );
+  });
+  const isSelected = !props.outsideCurrentMonth && !!matchingDay;
+  return (
+    <Badge
+      key={props.day.toString()}
+      overlap="circular"
+      badgeContent={
+        isSelected ? (
+          <Tooltip title={matchingDay?.label ?? ''}>
+            <span>⌛</span>
```

Comment:
I would have thought that an alert indicator is more appropriate?
```suggestion
            <span>⚠️</span>
```

or a _stop_ or _empty_ sort of thing.